app/web_hub.go

```diff
@@ -129,19 +125,20 @@ func (a *App) HubStop() {
 
 // GetHubForUserId returns the hub for a given user id.
 func (s *Server) GetHubForUserId(userId string) *Hub {
-	if len(s.GetHubs()) == 0 {
+	// TODO: This check is probably redundant and can be removed.
```

Comment:
Then can we remove it?

Reply:
Yeah I think so. I just need to ensure that this method is _never_ called before the hub is initialized. I wanted to perhaps delay that investigation to a later point, but no harm in spending some more time on it.